### DISCUSSION TITLE
Import "customtkinter" could not be resolved

### DISCUSSION THREAD

**Discussion by totallynotdrait** [TomSchimansky/CustomTkinter#700] [General]
so I installed customtkinter on my 2014 MacBook Air but when i run the python script with the module it gives me this error

![Schermata 2022-11-30 alle 19 25 51](https://user-images.githubusercontent.com/108739871/204878579-ddcffc0b-8880-47ad-8108-8de86afa6391.png)

**Comment by Akascape** [comment]
This discussion may help:
https://github.com/TomSchimansky/CustomTkinter/discussions/636

**Comment by learnerAbhishek** [comment]
Hi @totallynotdrait .
I think customtkinter may not happen installed properly.

Can you please run "pip list" and check if you can find customtkinter in the list?

Thank you.

**Reply by totallynotdrait** [reply]
hey @learnerAbhishek 
i did `pip list` in my terminal and apparently there is customtkinter

![Schermata 2022-11-30 alle 23 36 39](https://user-images.githubusercontent.com/108739871/204923644-aa30eb08-f3f0-464f-9031-a5932b2be972.png)
